www/app/App.js

```diff
@@ -26,7 +25,17 @@ export class App extends NextApp {
                     <title>{ SEO_DATA.title }</title>
                     <meta name="description" content={ SEO_DATA.description } />
                     <meta name="keywords" content={ SEO_DATA.keywords.join(' ') } />
-                    <link rel="shortcut icon" type="image/x-icon" href={ favicon } />
+                    <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png?v=M4KN2GElyG" />
```

Comment:
Since we are no longer using webpack for all of the files, we can instead add this to the `Document.js`. I think it makes more sense.